atd-vze/src/Components/TableWithFilters.js

```diff
@@ -98,12 +129,17 @@ const TableWithFilters = ({
     tableQuery,
     defaultQuery,
     filterQuery,
+    dateRangeFilter,
   ]);
 
   const { loading, error, data } = useQuery(
     gql`
       ${tableQuery}
-    `
+    `,
+    // Disables cache as a workaround for loading never returning to false
+    // after queries with long time spans
+    // https://github.com/apollographql/react-apollo/issues/3361
```

Comment:
👀

Reply:
I added some comments about an alternate workaround that Sergio came up with so we can keep that in the equation down the line as well.